packages/p2p/src/stores/general-store.js

```diff
@@ -340,6 +344,11 @@ export default class GeneralStore extends BaseStore {
         this.is_loading = is_loading;
     }
 
+    @action.bound
+    setIsPaymentAgentBlocked(is_p2p_blocked_for_pa) {
```

Comment:
Your set function should be named the same as your observable, so this should be `setIsP2pBlockedForPa`